modules/features/endofyear/src/main/java/au/com/shiftyjelly/pocketcasts/endofyear/ui/StoriesPage.kt

```diff
@@ -103,9 +92,24 @@ internal fun StoriesPage(
         }
 
         if (state is UiState.Failure) {
-            ErrorMessage(onRetry)
-        } else if (state is UiState.Syncing || !isTextSizeComputed) {
-            LoadingIndicator()
+            onFailedToLoad()
+        } else if (!isTextSizeComputed) {
```

Comment:
❓ From my understanding this variable no longer has any use. Last year it was needed to compute the correct size to fit the text. I think we can remove it now and all the calculations associated with it.